.gitignore

```diff
@@ -1,4 +1,8 @@
 bin/
 test/out/img.tar
 **/target/
-!src/bin/
\ No newline at end of file
+!src/bin/
+
+# IDE related ignores
```

Comment:
In the core projects we've generally kept IDE-related ignores out and encouraged people to use a global gitignore file on their local machine, but I don't think we've made that a hard and fast rule. The downside to including them here is that the list can keep growing essentially unbounded as more contributors use different editors.